stable/mariadb/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 4.0.1
+version: 4.0.2
```

Comment:
according to this similar PR it seems it is better to update the minor version. So use `4.1.0` instead

Reply:
done